Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -45,7 +45,7 @@ python3 create-external-cluster-resources.py --rbd-data-pool-name <pool_name> --
 - `--skip-monitoring-endpoint`: (optional) Skip prometheus exporter endpoints, even if they are available. Useful if the prometheus module is not enabled
 - `--ceph-conf`: (optional) Provide a Ceph conf file
 - `--keyring`: (optional) Path to Ceph keyring file, to be used with `--ceph-conf`
-- `--cluster-name`: (optional) Ceph cluster name
+- `--k8s-cluster-name`: (optional) Kubernetes cluster name
```

Comment:
If someone is still using the `--cluster-name` flag will they get an error? It will be good to make it obvious that they need to rename the parameter.

Reply:
No there won't be any error,

The script is used at very start just to connect the cluster.

Reply:
Only during upgrade caps they need to take care of it to use new flag, 

But which is already updated with the script description

Reply:
Could the script print an error if `--cluster-name` is passed? The error can tell them to instead use `--k8s-cluster-name`. This way, if someone doesn't look at the release notes they can quickly know they have need to rename this parameter.

Reply:
So if we use `--cluster-name` it will print all the correct flags that can be used

Reply:
Should we mention this in release notes?